src/abi/MicroCreditABI.json

```diff
@@ -0,0 +1,273 @@
+[
```

Comment:
You are only using `userLoan`, `claimLoan`, and `repayLoan`. You can remove the rest, no need to have the complete ABI.